tests/unit/OrgPerson.spec.ts

```diff
@@ -183,9 +181,9 @@ function createComponent (
   })
 }
 
-describe('Org/Person component for Correction', () => {
+describe('Org/Person component for a Correction filing', () => {
   beforeAll(() => {
-    store.state.stateModel.tombstone.filingType = FilingTypes.CORRECTION
```

Comment:
Tests should hard-code values to make them more fragile. Otherwise an error in the enum would not be caught here.